tests/snappi/conftest.py

```diff
@@ -76,3 +76,29 @@ def enable_packet_aging_after_test(duthosts, rand_one_dut_hostname):
 
     duthost = duthosts[rand_one_dut_hostname]
     enable_packet_aging(duthost)
+
+
+@pytest.fixture(scope="module")
+def pfc_pause_delay_values(duthosts, rand_one_dut_hostname):
```

Comment:
It seems to me that the name is a little bit misleading. How about pfc_process_delay_test_params?